stacks-core/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+use thiserror::Error;
+
+pub mod address;
+pub mod c32;
+pub mod crypto;
+pub mod utils;
+
+#[derive(Error, Debug, Clone)]
+pub enum StacksError {
+    #[error("Invalid arguments: {0}")]
+    InvalidArguments(&'static str),
+    #[error("Could not crackford32 encode or decode: {0}")]
+    C32Error(#[from] c32::C32Error),
+}
+
+pub type StacksResult<T> = Result<T, StacksError>;
+
+pub mod prelude {
+    pub use super::StacksError;
+    pub use super::StacksResult;
+
+    pub use crate::address::*;
+    pub use crate::c32::*;
+    pub use crate::crypto::*;
+    pub use crate::utils::*;
+}
```

Comment:
Nit: Personally I'd like to avoid wildcard imports and big preludes. I generally prefer explicit imports.

Reply:
From my experience preludes can be very handy when working with libraries. It decreases friction even further for beginners.

Let me know if you want to veto it and I'll remove it - no strong preferences from me.